src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,527 @@
+                            setGraphic(IconTheme.JabRefIcon.DELETE_ENTRY.getGraphicNode());
+                            setOnMouseClicked(evt -> removeAbbreviation());
+                        }
+                    }
+                } else {
+                    setGraphic(null);
+                }
+            }
+
+        });
+    }
+
+    private void setBindings() {
+        viewModel.currentFileProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalFilesBox.getSelectionModel().select(newvalue);
+            if (newvalue != null) {
+                isEditableAndRemovable.set(!newvalue.isBuiltInListProperty().get());
+            }
+            removeJournalAbbreviationsButton.setDisable((newvalue == null) || !isEditableAndRemovable.get());
+        });
+        viewModel.currentAbbreviationProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalAbbreviationsTable.getSelectionModel().select(newvalue);
+        });
+        journalAbbreviationsTable.getSelectionModel().selectedItemProperty()
+                .addListener((observable, oldvalue, newvalue) -> {
+                    viewModel.currentAbbreviationProperty().set(newvalue);
+                });
+        journalFilesBox.itemsProperty().addListener((observable, oldvalue, newvalue) -> {
+            viewModel.journalFilesProperty().set(newvalue);
+        });
+        journalFilesBox.getSelectionModel().selectedItemProperty().addListener((observabe, oldvalue, newvalue) -> {
+            viewModel.changeActiveFile(newvalue);
+        });
+        viewModel.journalFilesProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalFilesBox.itemsProperty().set(newvalue);
+        });
+        viewModel.abbreviationsProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalAbbreviationsTable.setItems(newvalue);
+        });
+        journalAbbreviationsTable.itemsProperty().addListener((observable, oldvalue, newvalue) -> {
+            viewModel.abbreviationsProperty().set(newvalue);
+        });
+        isEditableAndRemovable.addListener((observable, oldvalue, newvalue) -> {
+            removeJournalAbbreviationsButton.setDisable(newvalue.booleanValue());
+        });
+    }
+
+    public void showAndWait() {
+        FXAlert journalAbbreviationsDialog = new FXAlert(AlertType.INFORMATION,
+                Localization.lang("Journal abbreviations"));
+        journalAbbreviationsDialog.setResizable(true);
+        journalAbbreviationsDialog.setDialogPane((DialogPane) this.getView());
+        ((Stage) this.getView().getScene().getWindow()).setMinHeight(400);
+        ((Stage) this.getView().getScene().getWindow()).setMinWidth(600);
+        journalAbbreviationsDialog.showAndWait();
+    }
+
+    @FXML
+    private void addNewFile() {
+        FileChooser chooser = new FileChooser();
+        chooser.getExtensionFilters().add(new ExtensionFilter(Localization.lang("%0 file", "TXT"), "*.txt"));
```

Comment:
Have a look at the new FileExtensions Enum I created in #1336. Already inlcudes the Loalization.